pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -82,9 +84,14 @@ func (configgen *ConfigGeneratorImpl) buildGatewayListeners(env model.Environmen
 			log.Warnf("Multiple servers on same port is not supported yet, port %d", server.Port.Number)
 			continue
 		}
+		var opts buildListenerOpts
+		var listenerType plugin.ListenerType
+		var networkFilters []listener.Filter
+		var hTTPFilters []*http_conn.HttpFilter
```

Comment:
Do we need these vars? Just declare them inline in the CallbackListenerMutableObjects below.

Reply:
yes, we do. they are passed by ptr and shared with marshalFilters.

Reply:
Ah, I missed that; it's fine. Also, small nit: `httpFilters`, not `hTTPFilters`

Reply:
i hate this but https://github.com/golang/go/wiki/CodeReviewComments#initialisms. http is high on the watch list. that necessarily leads to local vars looking strange like this. we can relax this rule for pilot i suppose (it's already a mix) but that's the default for go.

Reply:
What you posted agrees with what I said:

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

It should be `httpFilters`, not `hTTPFilters` as it is in the PR. This is because initialism should have a consistent case and visibility requires that it the first letter be lower case, therefore the entire initialism should be lower case.

Reply:
hmm, i wonder if this changed at some point. i could swear i've had to do these silly caps to make some lint check pass. anyway, done.